submissions/onyinyeChukwuka/HTML-Forms/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```

Comment:
@Onyinye101 You don't need to create a new product page. You only need to link the already created product page to contactform page and vice versa

Reply:
Thank you, I have corrected it now.